config/webpack.common.js

```diff
@@ -178,6 +179,17 @@ module.exports = function (options) {
        * See: https://github.com/webpack/docs/wiki/optimization#multi-page-app
        */
       new CommonsChunkPlugin({
```

Comment:
I don't get what this instance of CommonsChunkPlugin is supposed to do.. As far as I can tell, with this settings it won't do anything at all. @colinskow could you elaborate please?